### PR TITLE
update set-clipboard

### PR DESCRIPTION
**Pull request by felenk**
adding additional example to address commonly needed task of adding ssh key to github via clipboard

# PR Summary

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [ ] Preview content
- [ ] Version 7.1 content
- [ ] Version 7.0 content
- [ ] Version 5.1 content

## PR Checklist

- [ ] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [ ] PR has a meaningful title
- [ ] PR is targeted at the _staging_ branch
- [ ] All relevant versions updated
- [ ] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [ ] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

**Review by sdwheeler — CHANGES_REQUESTED** [review]
@felenk Thanks for the suggestion. We could use an example that shows pipeline input like this but I am not sure how this example "adds the key to GitHub".

I would reword the title of the example to be "Copy the contents of a file to the clipboard". Then add an introductory sentence to describe why you would want to use this example.

If you update this PR, please update all version of this article (5.1,7.0, and 7.2). For more information, see https://aka.ms/PSDocsContributor.

**Comment by felenk** [comment]
Good thinking

**Review by felenk — COMMENTED** [review]
Good idea

**Comment by sdwheeler** [comment]
The PR is replaced by #6975
